docs/_includes/common/dialog.html

```diff
@@ -207,6 +207,63 @@ <h2 id="dialog-title-default" class="large-text-1 bold-text">Heading</h2>
 </div>
     {% endhighlight %}
 
+    {% if page.ds == 6 %}
+    <h3>Right Panel Dialog with header link (DS6 only)</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <button class="btn btn--primary dialog-button" data-makeup-dialog-button-for="dialog-right-panel-1" type="button">Preview</button>
+            <div aria-labelledby="dialog-title-right" aria-modal="true" class="dialog" hidden id="dialog-right-panel-1" role="dialog">
+                <div class="dialog__window dialog__window--right">
+                    <div class="dialog__header">
+                        <button aria-label="Close dialog" class="dialog__close" type="button">
+                            <svg aria-hidden="true" class="icon icon--close" focusable="false" height="16" width="16">
+                                <use xlink:href="#icon-close"></use>
+                            </svg>
+                        </button>
+                        <h2 id="dialog-title-default" class="dialog__title dialog__title--center large-text-1 bold-text">Heading</h2>
+                        <a class="dialog__end-text">
```

Comment:
This isn't keyboard focusable. This should be a button tag, right?

Reply:
I changed it to button and used secondary with no border to match the color, although links should be focusable if you enable that option in safari.

Reply:
Thanks. p.s. links always need an href to be keyboard focusable.